modules/compliance-anatomy.adoc

```diff
@@ -0,0 +1,284 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-troubleshooting.adoc
+
+[id="compliance_anatomy_{context}"]
+= Anatomy of a scan
+
+The following sections outline the components and stages of Compliance Operator scans.
+
+== Compliance sources
+The compliance content is stored in `Profile` objects that are generated from a `ProfileBundle`. The Compliance Operator creates a `ProfileBundle` for the cluster and another for the cluster nodes.
+
+[source,terminal]
+----
+$ oc get profilebundle.compliance
+$ oc get profile.compliance
+----
+
+The `ProfileBundle` objects are processed by deployments labeled with the `Bundle` name. To troubleshoot an issue with the `Bundle`, you can find the deployment and check out logs of the pods in a deployment:
+
+[source,terminal]
+----
+$ oc logs -lprofile-bundle=ocp4
+$ oc get deployments,pods -lprofile-bundle=ocp4
+$ oc logs pods/...
```

Comment:
can we add `$pod_name` flag `-c` with `$container_name` if this is specific to profileparser pods 
`$ oc describe pod/$pod_name -c $container_name`